kolibri/plugins/coach/assets/src/views/common/notifications/NotificationCard.vue

```diff
@@ -54,7 +54,7 @@
     mixins: [commonCoach],
     props: {
       targetPage: {
-        type: String,
+        type: Object,
```

Comment:
FYI, we have a validator than would be helpful here:

```javascript
import { validateLinkObject } from 'kolibri.utils.validators';
```

not necessary for now